codalab/lib/upload_manager.py

```diff
@@ -176,36 +176,6 @@ def _simplify_directory(self, path, child_path=None):
         path_util.rename(child_path, path)
         path_util.remove(temp_path)
 
-    def update_metadata_and_save(self, bundle, enforce_disk_quota=False):
```

Comment:
Moved this function to `bundle_model` because otherwise calculating and saving the disk use of shared filesystem bundles would be very difficult.
I also think this kind of metadata calculation makes a lot more sense to be in a less specific module than `upload_manager` in a world where bundles could be created without being uploaded.